src/Algebra/Graph.hs

```diff
@@ -261,7 +261,11 @@ ordIntR x y = compare (toAdjacencyIntMap x) (toAdjacencyIntMap y)
 
 instance Applicative Graph where
     pure  = Vertex
-    (<*>) = ap
+    (<*>) = seqApR
+
+seqApR :: Graph (a -> b) -> Graph a -> Graph b
```

Comment:
What does the `seq` prefix mean? I think `apR` would be clearer and more consistent with other such functions.

Reply:
`seqAp` for "sequential application", but you are right, it can be confused with `seq` :p